plugins/gcp/resoto_plugin_gcp/collector.py

```diff
@@ -547,6 +553,7 @@ def collect_something(
         successors: List = None,
         predecessors: List = None,
         client_kwargs: Dict = None,
+        client_nested_callables: Type[str] = [],
```

Comment:
Should this be `List[str]`?

If `client_nested_callables` should default to an empty list the better pattern to use is
```
client_nested_callables: List[str] = None,
if client_nested_callables is None:
    client_nested_callables = []
```

Otherwise every instance of this class will share the same reference to that empty list and if one manipulates it it changes in all the other object instances as well.

Reply:
Makes sense, thanks!